.github/workflows/docker_dealii.yml

```
@@ -0,0 +1,52 @@
+name: Docker with deal.II
+
+on:
+  workflow_dispatch:
```

```suggestion
on:
  workflow_dispatch:
  workflow_run:
    workflows: [docker]
    types: [completed]
```
You could add this to automatically build the dealii image after the docker workflow built the dependencies image. (Please check if this makes sense)